framework/test/jest/integration/specs/modules/chain/blocks/chain/pop_last_block.spec.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+describe('integration test (blocks) - chain/popLastBlock', () => {
+	describe('popLastBlock', () => {
```

Comment:
Maybe simplify the level of describes here so we have less levels?

Reply:
It follows the structure of the code, anyone who implement and refactor the code as well can improve it.